src/sass/components/newCampaignForm.scss

```diff
@@ -0,0 +1,49 @@
+@import 'sass/helpers/colors.scss';
+
+header {
+  padding: 10px;
+}
+.ant-form-item-control {
```

Comment:
If you're going to override .ant why not define these inside an `and-overrides.scss` file instead?